src/controllers/travisauth.controller.js

```diff
@@ -34,9 +34,9 @@ export async function addTravisTokenForUser(req, res, next) {
   }
 
   const userId = req.user.id;
-  const token = req.body.travisToken;
+  const githubToken = req.body.githubToken; // travis exchanges github token for travis token
```

Comment:
This githubToken does not exist/is not used

Reply:
this isn't needed, I'll remove it